Assets/_System/Scripts/TurnBasedScripts/BaseUnitClass.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public enum UnitType { Swordsman}
+
```

Comment:
Wrap this in a namespace. Something that sounds appropriate. 

DogHouse.ToonWorld.WhateverYouThinkTheNamespaceShouldBeCalled

Maybe like Unit or something.

Reply:
Wrapped all my code in the Doghouse.ToonWorld.CombatControllers